packages/core/scripts/umip-3/2_VoteSimulate.js

```diff
@@ -22,10 +23,19 @@ const foundationWallet = "0x7a3A1c2De64f20EB5e916F40D11B01C441b2A8Dc";
 // Use the same ABI's as deployed contracts:
 const { getTruffleContract } = require("../../index");
 const Governor = getTruffleContract("Governor", web3, "1.1.0");
+// Use VotingInterface when committing or revealing to match the correct commit and reveal function signatures
+// without an `ancillaryData` param.
+const VotingInterface = getTruffleContract("VotingInterface", web3);
```

Comment:
Main change here: We use the `VotingInterface` ABI since it doesn't overload the `commitVote` function signature